change_events_test.go

```diff
@@ -81,12 +76,7 @@ func TestChangeEvent_CreateWithPayloadVerification(t *testing.T) {
 		},
 	)
 
-	client := &Client{
-		v2EventsAPIEndpoint: server.URL,
-		apiEndpoint:         server.URL,
-		authToken:           "foo",
-		HTTPClient:          defaultHTTPClient,
-	}
+	client := defaultTestClient(server.URL, "foo")
```

Comment:
Technically, shouldn't these also be passing values for the `v2EventsAPIEndpoint`?

Reply:
It does: https://github.com/PagerDuty/go-pagerduty/pull/325/files/4f01c5befe6bde8e812f8acf42546764d58bf7c8#diff-4dc56e64f310cd8c12bb00f75c09e83f5d875563ea9901788e525487dc6788b3R41